### ISSUE TITLE
does post can auto fire?

### ISSUE THREAD

**Issue by bbaduser** [ava/use-http#382]
![image](https://user-images.githubusercontent.com/120174736/210047770-c02a6143-366f-4fd8-9574-152f46c521a9.png)
can anyone tell me , the post fetch can run onmount or onupdate ?

**Comment by krishnasaga** [comment]
@bbaduser You can do it by calling the function `post' function from `useEffect`


```javascript

const { loading,error,data = [] } = useFetch('...',options,[]);

useEffect(()=>{
  post();
},[]);

```

**Comment by iamthesiz** [comment]
Yes, `const { data, loading } = useFetch('url', { method: 'post' }, [])`

**Comment by TannerS** [comment]
> Yes, `const { data, loading } = useFetch('url', { method: 'post' }, [])`

Then how do you pass in the post body?
